### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by ngeelani48**
### In this Pull Request, the Following SQL commands are included with corresponding screenshots:

**- Add a column species of type string to the animals' table.**
Screenshot (58)

**- Insert data of new animals into the table.**
Screenshot (59)

**- Update the animals' table by setting the species column to unspecified.**
Screenshot (60)

**- Roll back the change and verify that the species columns went back to the state before the transaction.**
Screenshot (61)

**- Update the animal's table by setting the species column to `digimon` for all animals that have a name ending in `mon` and setting the species column to `pokemon` for all animals that don't have species already set.**
Screenshot (62)

**- Inside a transaction delete all records in the animal's table, then roll back the transaction.**
Screenshot (63)

**- Inside a transaction:**
  - Delete all animals born after Jan 1st, 2022.
  - Create a savepoint for the transaction.
  - Update all animals' weight to be their weight multiplied by -1.
Screenshot (64)

  - Rollback to the savepoint
  - Update all animals' weights that are negative to be their weight multiplied by -1.
  - Commit transaction
Screenshot (65)

**- Write queries to answer the questions:**
  - How many animals are there?
Screenshot (66)

  - How many animals have never tried to escape?
Screenshot (67)

  - What is the average weight of animals?
Screenshot (68)

  - Who escapes the most, neutered or not neutered animals?
Screenshot (69)

  - What is the minimum and maximum weight of each type of animal?
Screenshot (70)

  - What is the average number of escape attempts per animal type of those born between 1990 and 2000?
Screenshot (71)

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,2 +1,3 @@
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered,weight_kg) VALUES (1, 'Agumon', '02/03/20', 0, true, 10.23),(2, 'Gabumon', '11/15/18', 2, true, 8.0), (3, 'Pikachu', '01/7/21', 1,false,15.04), (4, 'Davimon', '05/12/17', 5,true,11);
 
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered,weight_kg) VALUES (5, 'Charmander', '02/08/20', 0, false, -11),(6, 'Plantmon', '11/15/21', 2, true, -5.7), (7, 'Squirtle', '04/02/93',3 , false, -12.13), (8, 'Angemon', '06/12/05', 1, true, -45),(9, 'Boarmon', '06/07/05', 7, true, 20.4),(10, 'Blossom', '10/13/98', 3, true, 17),(11, 'Ditto', '05/14/22', 4, true, 22);
```
Comment (Qoosim) [inline data.sql]: - [ ] I got this error message while trying to populate your table with the data inside the data.sql. Kindly look into it and fix.

### Error Message
postgres

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @ngeelani48, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  Good README file. :heavy_check_mark: 
-  Correct Git flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (ngeelani48) [reply data.sql]: @Qoosim Thanks for the review and your time 👍 . It was working on my side but I have made the changes and hope this time it gets accepted. 🤞 
Thanks Again. 💮 
Screenshot (72)

**Review by danmainah — APPROVED** [review]
Hi @ngeelani48 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️  Good design of the database and queries
✔️  Good PR and Description

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**